build.gradle

```diff
@@ -542,27 +545,59 @@ task bundle(type: Zip) {
         rename 'GATKConfig.properties', 'GATKConfig.EXAMPLE.properties'
     }
 
-    from("${buildDir}/${pythonPackageArchiveName}")
-    // When including gatkcondaenv.yml file in the release bundle, strip off the
-    // 'build/' prefix used for the location of the Python package archive.
-    from("scripts/gatkcondaenv.yml", {
-        filter { line -> line.replace("build/${pythonPackageArchiveName}", pythonPackageArchiveName) }
-    })
+    from("$buildDir/$pythonPackageArchiveName")
+    from("$buildDir/$gatkCondaYML")
+    from("$buildDir/$gatkCondaIntelYML")
     into(baseName)
 
     doLast {
         logger.lifecycle("Created GATK distribution in ${destinationDir}/${archiveName}")
     }
 }
 
-task createPythonPackageArchive(type: Zip) {
+// Create the python package archive file, and two GATK conda environment yml files from the template
+// (one for standard GATK and one for running GATK with Intel hardware).
+task condaEnvironmentDefinition() {
+    dependsOn 'pythonPackageArchive'
+    inputs.file gatkCondaTemplate
+    outputs.dir buildDir
```

Comment:
is it weird to have buildDir as the output?  Shouldn't it just be the two output files?

Reply:
Yes. I removed these input/output declarations altogether since this task now exists only as a dependency aggregator.